File: SecondYear/gamandeepsingh_GamandeepSingh_2226csit1042_2/Week_3/SortingSentence.java
class Solution {
    public String sortSentence(String s) {
        String[] words = s.split(" ");

        String[] originalWords = new String[words.length];

        for (String word : words) {
            int index = Integer.parseInt(word.substring(word.length() - 1)) - 1;
            originalWords[index] = word.substring(0, word.length() - 1);
        }

        StringBuilder originalSentence = new StringBuilder();
        for (String word : originalWords) {
            originalSentence.append(word).append(" ");
        }

        return originalSentence.toString().trim();
    }
}

File: SecondYear/praptisharma28_PraptiSharma_2226CSE(AI&ML)1008_2/Week_1/Week_3/sortingsentence.py
class Solution:
    def sortSentence(self, s: str) -> str:
        words = s.split()

        originalWords = [None] * len(words)

        for word in words:
            index = int(word[-1]) - 1
            originalWords[index] = word[:-1]

        originalSentence = " ".join(originalWords)

        return originalSentence
